p4src/linux_networking/README_LINUX_NETWORKING_WITH_ECMP.md

```diff
@@ -216,17 +238,55 @@ Note: Port number used in p4rt-ctl commands are target datapath indexes(unique i
     p4rt-ctl add-entry br0 linux_networking_control.handle_tx_control_pkts_table "istd.input_port=5,action=linux_networking_control.set_control_dest(4)"
   ```
 
-#### 14. Configure ECMP for underlay connectivity
+#### 15. Configure ECMP for underlay connectivity
 - **Rule for: To reach link partner multiple paths are configured, packets can be hashed to any port where the nexthop's ARP is learnt**
```

Comment:
Why are these lines in bold face? You generally don't want to emphasize this much text at one time.